CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 5.X.Y
 * Feat: Replaced `punycode` package usage with `url.domainToASCII` (#630).
 * Feat: World of Padman (2007) - Added support (#636)
+* Feat: Renegade X (2014) - Added support (#643)
```

Comment:
You could also mention yourself here if you'd like (:

Reply:
It's fine without